src/RoleObjects/AccessibilityObject.js

```diff
@@ -177,6 +177,117 @@ export default class AccessibilityObject {
     return this._reactProps.accessKey;
   }
 
+  /**
+   * The translate attribute specifies whether the content of an element should be translated or not.
+   * @access public
+   * @param {String} str - "yes" or "no" Specifies that the content of the element should be translated or not
+   */
+  set translate(str) {
+    this._reactProps.translate = str;
+  }
+
+  /**
+   * The translate attribute specifies whether the content of an element should be translated or not.
+   * @access public
+   * @returns {String} string containing the letter to use for the keyboard shortcut
+   */
+  get translate() {
+    return this._reactProps.translate;
+  }
+
+  /**
+   * The spellcheck attribute specifies whether the element is to have its spelling and grammar checked or not.
+   * @access public
+   * @param { boolean } bool
+   */
+  set spellcheck(bool) {
+    this._reactProps.spellCheck = bool;
+  }
+
+  /**
+   * The spellcheck attribute specifies whether the element is to have its spelling and grammar checked or not.
+   * @access public
+   * @returns {boolean} boolean: true or false
+   */
+  get spellcheck() {
+    return this._reactProps.spellCheck;
+  }
+
+  /**
+   * The contenteditable attribute specifies whether the content of an element is editable or not.
+   * @access public
+   * @param { boolean } bool true or false
+   */
+  set contenteditable(bool) {
+    this._reactProps.contentEditable = bool;
+  }
+
+  /**
+   * The contenteditable attribute specifies whether the content of an element is editable or not.
+   * @access public
+   * @returns {boolean} true or false
+   */
+  get contenteditable() {
+    return this._reactProps.contentEditable;
+  }
+
+  /**
+   * Sets the title for the DisplayObject
+   * @access public
+   * @param {String} title - string mouse hover show text of title as tooltip
+   */
+  set title(title) {
+    this._reactProps.title = title;
+  }
+
+  /**
+   * Retrieves the title for the DisplayObject
+   * @access public
+   * @returns {String} string mouse hover show text of title as tooltip
+   */
+  get title() {
+    return this._reactProps.title;
+  }
+
+  /**
+   * The lang attribute specifies the language of the element's content.
+   * @access public
+   * @returns {String} lang: eg. "en" for English, "es" for Spanish, "fr" for French, and so on.
```

Comment:
Nit: please include in the comment that https://www.w3schools.com/tags/ref_language_codes.asp has the full list of valid values.

Reply:
updated